src/runtime.js

```diff
@@ -78,82 +78,173 @@ define([
         var nodeType = ast.type;
         var result = defaultResult;
 
-        // TODO order alphabetically
         switch (nodeType) {
-        case "Program":
-            result = this.visitProgram(ast, state);
+        case "ArrayExpression":
+            result = this.visitArrayExpression(ast, state);
             break;
-        case "ExpressionStatement":
-            result = this.accept(ast.expression, state);
+        case "ArrowExpression":
+            console.log("ArrowExpression visitor not implemented yet");
```

Comment:
the main objective of my pull request is to order all visitors (missing or not) in alphabetic order. Your solution is more elegant but it will unorder the code again.

But I will change console.log to console.warn as you suggest